src/main/groovy/org/pih/warehouse/api/StockMovementItem.groovy

```diff
@@ -380,11 +380,11 @@ class AvailableItem {
 enum AvailableItemStatus {
     AVAILABLE, PICKED, RECALLED, HOLD, NOT_AVAILABLE
 
-    static getList() {
+    static list() {
```

Comment:
have you made sure it is not used anywhere with the `getList()` version?

Reply:
Yep I am sure, It was me who introduced this variable in the last PR